styles.css

```diff
@@ -0,0 +1,101 @@
+@import url('https://fonts.googleapis.com/css2?family=Archivo:wght@300;400;600&display=swap');
+
+* {
+    --font-family: 'Archivo', sans-serif;
+    font-family: 'Archivo', sans-serif;
+    --bg:#ff66c4;
+    --input-bg:#d9d9d9;
+    --btn:#5e17eb;
+}
+
+body{
+    background-color: var(--bg);
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+}
+
+.container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    padding: 3rem 1.5rem;
+}
+
+.wrapper{
+    flex-grow: inherit;
+    margin-top: 2rem;
+}
+
+.sect{
+    margin-top: 2rem;
+}
+
+h1{
+    line-height: .8;
+    /* for media queries */
+    font-size: clamp(2em, 3vw, 2.5rem);
+}
+
+h3{
+    line-height: .5;
+    font-size: clamp(1.05em, 1.5vw, 1.5rem);
```

Comment:
What does `clamp` do?

Reply:
It's a CSS method used for responsiveness in general. It takes in 3 parameters (the first in this case is the minimum font size, the second parameter is the preferred font size and the last is the maximum font size). Attached is an article that explains more about [clamp](https://developer.mozilla.org/en-US/docs/Web/CSS/clampl)